_posts/2016-07-29-npm-install-electron.md

```diff
@@ -0,0 +1,67 @@
+---
+title: npm install electron
+author: zeke
+---
+
+With help from the community, we have taken over the `electron` package
+name on npm. As of Electron version 1.3.1, you can
+`npm install electron --save-dev` to install the latest precompiled version
+of Electron in your app.
+
+---
+
+![npm install electron](https://cloud.githubusercontent.com/assets/378023/17259327/3e3196be-55cb-11e6-8156-525e9c45e66e.png)
+
+## The History of Electron Prebuilt
+
+If you've ever worked on an Electron app before, you've likely come across the
+`electron-prebuilt` npm package. This package is an indispensable part of nearly
+every Electron project. When installed, it detects your operating system
+and downloads a prebuilt binary that is compiled to work on your system's
+architecture.
+
+## The Name
+
+The Electron installation process was often a stumbling block for new users.
+Many brave people tried to get started without reading the documentation, and
+would often run `npm install electron` instead of `npm install electron-prebuilt`,
+only to discover (often after much confusion) that it was not the `electron`
+they were looking for.
+
+The reason for this is that there was an existing `electron` project on npm,
+created before GitHub's Electron project even existed. To help make Electron
+development easier and more intuitive for new users, we reached out to the
+owner of the existing `electron` package to ask if he'd be willing to let us use
+the name. Luckily he was a fan of our project, and agreed to help us repurpose
+the name.
+
+## Prebuilt Lives On
+
+As of version `1.3.1`, we have begun publishing `electron` and `electron-prebuilt`
```

Comment:
Not sure where to put this but it might be helpful to link somewhere to https://www.npmjs.com/package/electron just so people can see it there.